lua/regexplainer/buffers/init.lua

```diff
@@ -77,6 +77,7 @@ function M.get_buffer(options)
         padding = { 1, 2 },
         style = 'shadow',
       },
+      win_options = (options and options.popup and options.popup.win_options) or {},
```

Comment:
this will need to take the shared options

Reply:
It seems to take this into account:

With winblend:
![image](https://user-images.githubusercontent.com/62098008/157566756-d8663ae1-b6d2-40fb-9ffb-c99948634eaf.png) 

Default config:
![image](https://user-images.githubusercontent.com/62098008/157566852-ba49ec1d-697f-43d1-a102-3b080349ebd0.png)

I personally would have used `vim.tbl_deep_extend` for merging shared_options + extra options, with user options (to avoid the checks for nil fields).

Reply:
I'd be happy with that. Would you like to make this pr about merging user options instead? Just update README and make it a breaking commit e.g. 

```
feat!: merge user config

deeply merge user config for popup and split windows and buffers,
preferring user values over defaults
```